installers/python-ci/radiasoft-download.sh

```diff
@@ -38,15 +39,16 @@ python_ci_main() {
         chown -R vagrant: '$d'
         # POSIT: no interpolated vars in names
         trap 'chown -R "$o" "$d"' EXIT
-        su - vagrant <<EOF2
+        su - vagrant <<'EOF2'
             set -eou pipefail
-            set -x
             cd '$d'
             export GITHUB_TOKEN='${GITHUB_TOKEN:-}'
-            if (( $p )); then
-                pip uninstall -y pykern >& /dev/null || true
-                pip install git+https://github.com/radiasoft/pykern.git
-            fi
+            # POSIT: no spaces or specials in $p values
+            for x in ${p[*]}
```

Comment:
RS bash style is `for x in y; do` not `for x in y
do`.

Reply:
3d91f17e1d051fa7340c1c5aafd7819b1e38da6d